src/app/projects/services/projects.service.ts

```diff
@@ -251,7 +251,9 @@ export class ProjectsService implements OnDestroy {
           obj.organ?.ontologies?.map((organ) => organ.ontology_label) ?? [],
         technologies:
           obj.technology?.ontologies?.map((tech) => tech.ontology_label) ?? [],
-        cellCount: obj.cellCount,
+        // TODO: Remove usage of cellCount once the cellCount has been copied to content.estimated_cell_count
+        // GH issue : https://github.com/ebi-ait/dcp-ingest-central/issues/445
+        cellCount: obj.content.estimated_cell_count || obj.cellCount,
```

Comment:
I don't think you can use `||` here. Is it possible cell count can be `0`?

Reply:
Good point!

so the template has the following logic:
```
<td class="vf-table__cell">
    <span *ngIf="project.cellCount">{{ (project.cellCount | number) }}</span>
    <p *ngIf="!project.cellCount">-</p>
</td>
```
I would assume that 0 or undefined is treated as the same and `-` is displayed for both. If estimated_cell_count is 0 is cellCount will either have a real value or undefined. And eventually, we will make sure in the migration script that cellCount will be copied to estimated_cell_count.

So i think it's okie!